WikiWatershed/model-my-watershed
# MMW Branding Updates

## Overview

Update splash page according to instructions in #2741. There are other places where we see "Model My Watershed", such as the app title (seen in tabs), the About page, etc, but those updates will have to wait for another day (see https://github.com/WikiWatershed/model-my-watershed/issues/2741#issuecomment-391510021).

This could use some styling adjustments before being merged.

Connects #2741 

### Demo

|        | Before | After |
|--------|--------|-------|
| **MMW**    | ![image](https://user-images.githubusercontent.com/1430060/40688965-2550ffce-636e-11e8-832c-15575910d4d0.png) | ![image](https://user-images.githubusercontent.com/1430060/40688978-3000c51c-636e-11e8-851d-d0e2b21a548a.png) |
| **BiG-CZ** | ![image](https://user-images.githubusercontent.com/1430060/40688995-3ec3b050-636e-11e8-96e4-5b9842847e2f.png) | ![image](https://user-images.githubusercontent.com/1430060/40689004-47cf0adc-636e-11e8-961f-6ac3df773749.png) |

/cc @ajrobbins @jfrankl for visual review.

## Testing Instructions

* Check out this branch and `bundle --debug`
* Ensure the splash page text matches that given in the issue
